### ISSUE TITLE
Avoid `display: none` on selectable tile input field

### ISSUE THREAD

**Issue by emyarod** [carbon-design-system/carbon#1669]
Related: https://github.com/IBM/carbon-components-react/issues/1114, reported by @asthajain1 and @zachstowellibm

## Detailed description

In the Selectable Tile component, the radio button `input` has the CSS rule `display: none` causing an a11y failure. We should explore a different strategy to visually hide the input field
